content/en/blog/2020/steering-election-results/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "2020 Steering Committee Election Results"
+subtitle: Welcome to our newest Steering Committee members
+description: Announcing the four newest Istio Steering Committee members.
+publishdate: 2020-09-29
+attribution: "Istio Steering Committee"
+keywords: [istio,steering,governance,community,election]
+---
+
+Last month, we [announced a revision to our Steering Committee charter](../steering-changes/), opening up governance roles to more contributors and community members. The Steering Committee now consists of 9 proportionally-allocated Contribution Seats, and 4 elected Community Seats.
+
+We have now concluded our [inaugural election](https://github.com/istio/community/tree/master/steering/elections/2020) for the Community Seats, and we're excited to welcome the following new members to the Committee:
+
+- [Neeraj Poddar](https://github.com/istio/community/blob/master/steering/elections/2020/nrjpoddar.md) (Aspen Mesh)
+- [Zack Butcher](https://github.com/istio/community/blob/master/steering/elections/2020/zackbutcher.md) (Tetrate)
+- [Christian Posta](https://github.com/istio/community/blob/master/steering/elections/2020/ceposta.md) (Solo.io)
+- [Zhonghu Xu](https://github.com/istio/community/blob/master/steering/elections/2020/hzxuzhonghu.md) (Huawei)
+
+They join Contribution Seat holders from Google, IBM/Red Hat and Salesforce. We now have representation from 7 organizations on Steering, reflecting the breadth of our contributor ecosystem.
```

Comment:
s/representation/representatives ?

Reply:
I like representation better as it puts the focus on the organizations. The point of this statement is to highlight the corporate diversity.